web/src/js/actions.js

```diff
@@ -117,6 +117,13 @@ export var FlowActions = {
     },
     clear: function(){
         $.post("/clear");
+    },
+    download: () => window.location = "/flows/dump",
+    upload: (file) => {
+        var filereader = new FileReader();
+        filereader.file = file;
+        filereader.onload = (e) => {$.post("/flows/dump", e.target.result); e.preventDefault();};
+        filereader.readAsBinaryString(file);
     }
```

Comment:
i changed it to the fetch html5 api, but now i dont have the _xsrf cookie out of the box, i added it to the request but now i get the following warning message form tronado:

`WARNING:tornado.general:403 POST /flows/dump?_xsrf=2%7Ca7b2e0dc%7C0e4ba265b3d5d563622e7dbeb8945579%7C1464868825 (::1): XSRF cookie does not match POST argument`